Add changelog entry for AI Assistant (#104)
nginx/documentation: content/nginx-one/changelog.md

Stay up-to-date with what's new and improved in the F5 NGINX One Console.

> ## January 16, 2025

Reviewer: When we merge, we'll combine this with https://github.com/nginx/documentation/pull/90/files 

```suggestion
## January 20, 2025
```

> In the F5 NGINX One Console, you can now select lines from your configuration files, and then select **Explain with AI**. The NGINX One AI Assistant explains those lines based on the official NGINX documentation.

Reviewer 2: do we want prompt context as heading? similar to Certificates, etc? So maybe 

```
### AI Assistant
```

above this line?

Reviewer: > do we want prompt context as heading? similar to Certificates and Config Sync Groups, etc? So maybe
> 
Good catch

```suggestion
### AI Assistant

In the F5 NGINX One Console, you can now select lines from your configuration files, and then select **Explain with AI**. The NGINX One AI Assistant explains those lines based on the official NGINX documentation.
```

> In the Configuration Editor you can now highlight a portion of the configuration and click "Explain with AI" to have the NGINX One AI Assistant give an explanation and guidance. AI Assistant is trained on the official NGINX documentation.

Reviewer: We haven't mentioned the `Configuration Editor` in our docs. And I'm making an interpretation based on what I've done to "Explain with AI":

```suggestion
In the F5 NGINX One Console, you can now select lines from your configuration files, and then select **Explain with AI**. The NGINX One AI Assistant explains those lines based on the official NGINX documentation.
```
